doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+
+An Angular router configuration, in startup, may look like this:
+
+Alternatively, an Aurelia router is configured inside the viewmodel that uses it. Thus, Aurelia configuration may look like this:
+
+<code-listing heading="Aurelia Router Configuration">
+	<source-code lang="ES 2015/2016">
+		export class App {
+		  configureRouter(config, router) {
+		    config.title = 'Aurelia';
+		    config.map([
+		      { route: ['', 'welcome'], name: 'welcome',      moduleId: 'welcome',      nav: true, title: 'Welcome' },
+		      { route: 'users',         name: 'users',        moduleId: 'users',        nav: true, title: 'Users' }
+		    ]);
+		
+		    this.router = router;
+		  }
+		}
+	</source-code>
+</code-listing>
+
+> Note: Router Migration
+> Migration to Aurelia will be easier if your application uses Angular's new router; this router shares many similarities with the router that Aurelia uses, and the configuration will be easier to migrate.
+
+#### Removing `$scope`
+
+Angular made heavy use of an Angular-specific service named `$scope`; responsible for computed properties, change detection (through `$scope.watch()`), signalling (via `$scope.broadcast` and `$scope.emit`), and view-to-viewmodel binding, this service was often a staple of an Angular controller.
+
+A typical controller might yield something like this:
+
+<code-listing heading="Angular Controller Configuration">
+	<source-code lang="ES 2015/2016">
+		angular.controller('Users', ['$scope', '$http', function ($scope, $http) {
+			$scope.heading = 'Welcome to the Angular Navigation App!';	$scope.firstName = 'John';
+			$scope.lastName = 'Doe';
+			$scope.previousValue = $scope.fullName();
+			
+			$scope.fullName = function () {
+				return $scope.firstName + ' ' + $scope.lastName;
+			};
+			
+			$scope.submit = function () {
+				$scope.previousValue = $scope.fullName();
+				alert('Welcome, ' + $scope.fullName());
+			};
+		}]);
+	</source-code>
+</code-listing>
+
+In Aurelia, viewmodel properties are declared as properties on the viewmodel object.
+
+The Aurelia version of our typical Angular controller might look like this:
+
+<code-listing heading="Aurelia Controller Configuration">
+	<source-code lang="ES 2015/2016">		export class Welcome {
+		  heading = 'Welcome to the Aurelia Navigation App!';
+		  firstName = 'John';
+		  lastName = 'Doe';
+		  previousValue = this.fullName;
+		
+		  get fullName() {
+		    return `${this.firstName} ${this.lastName}`;
+		  }
+		
+		  submit() {
+		    this.previousValue = this.fullName;
+		    alert(`Welcome, ${this.fullName}!`);
+		  }
+		}
+	</source-code>
+</code-listing>
+
+> Note: Controller Migration
+> Later versions of Angular supported the "controller as" syntax, allowing developers to declare viewmodel properties on the `this` property. However, `$scope` was still necessary for watches and signalling.) Your application will be easier to migrate to Aurelia if you are only using `$scope` for signalling.
```

Comment:
I'd add an example for "controller as" syntax as well before showing corresponding Aurelia controller.